reference/7.5/Microsoft.PowerShell.Core/About/about_Type_Conversion.md

```diff
@@ -0,0 +1,497 @@
+Locale: en-US
+ms.date: 12/05/2024
+online version: https://learn.microsoft.com/powershell/module/microsoft.powershell.core/about/about_type_conversion?view=powershell-7.5&WT.mc_id=ps-gethelp
+schema: 2.0.0
+title: about_Type_Conversion
+---
+# about_Type_Conversion
+
+## Short description
+
+PowerShell has a flexible type system that makes it easier to use. However, you
+must understand how it works to avoid unexpected results.
+
+## Long description
+
+PowerShell has a flexible type system that makes it easier to use. However, you
+must understand how it works to avoid unexpected results.
+
+By default, PowerShell _variables_ aren't _type-constrained_. You can create a
+variable containing an instance of one type and later assign values of any
+other type. Also, PowerShell automatically converts values to other types, both
+explicitly and implicitly. While implicit type conversion can be helpful, there
+are pitfalls, especially for users more familiar with languages have stricter
+type handling.
+
+## Type-constrained variables and explicit types conversion
+
+To type-constrain a variable, place a type literal to the left of the
+variable name in an assignment. For example:
+
+```powershell
+[int]$foo = 42
+```
+
+You can use type casting to explicitly convert a value to a specific type. For
+example:
+
+```powershell
+PS> $var = [int]'43'
+PS> $var.GetType().Name
+Int32
+```
+
+Type casting ensures that only values of the specified type can be assigned to
+the variable. If you try to assign a value of a different type that can be
+converted to the constrained type, PowerShell performs an implicit conversion.
+For more information, see the [Implicit type conversion][03] section of this
+article.
+
+### Numeric type conversion
+
+Numeric types can be freely converted to one another as long as the target type
+is wide enough. For example:
+
+```powershell
+PS> (42.1).GetType().Name
+Double
+PS> $byte = [byte] 42.1
+PS> $byte
+42
+PS> $byte.GetType().Name
+Byte
+```
+
+The value `42.1` is a **Double**. When you convert it to a **Byte**, the value
+is truncated to an integer `42`, which is small enough to fit into a **Byte**.
+
+### Boolean type conversion
+
+A value of _any_ type can be coerced to a **Boolean**.
+
+- For numeric types, `0` converts to `$false` and any other value converts to
+  `$true`.
+
+  ```powershell
+  PS> [boolean]0
+  False
+  PS> [boolean]0.0
+  False
+  PS> [boolean]-1
+  True
+  PS> [boolean]1
+  True
+  PS> [boolean]42.1
+  True
+  ```
+
+- For other types, null values, empty strings, and empty arrays are converted
+  to `$false`. Other values, including empty hashtables convert to `$true`.
+
+  ```powershell
+  PS> [boolean]''
+  False
+  PS> [boolean]@()
+  False
+  PS> [boolean]'Hello'
+  True
+  PS> [boolean]@(1,'Hello')
+  True
+  PS> [boolean]@{}
+  True
+  ```
+
+### String type conversion
+
+A value of _any_ type can be coerced to a **String**. The default conversion is
```

Comment:
**String** is capitalized and in bold in this sentence. It's in lower case and not bold in the next paragraph. Is this styling correct?